conda_recipes/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM nvidia/cuda:8.0-cudnn6-devel-ubuntu14.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update
+
+RUN apt-get install -y --no-install-recommends make git ssh realpath wget unzip cmake3 vim
+RUN apt-get install -y --no-install-recommends libgoogle-glog-dev libyaml-dev
+RUN apt-get install -y --no-install-recommends libgtest-dev libz-dev libgmp3-dev
+RUN apt-get install -y --no-install-recommends automake libtool valgrind subversion
+RUN apt-get install -y --no-install-recommends ca-certificates software-properties-common
+
+RUN cmake --version
+
+# GCC 4.8.*
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends libcilkrts5 gcc-4.8 g++-4.8
```

Comment:
could you explain why it is required to install ```gcc-4.8``` manually? 
IIRC ubuntu 14.04 (trusty) should ship with ```gcc-4.8```.

Reply:
It should but this is done as a safety measure.

Reply:
#resolved